lib/mutant/mutator/node/literal/array.rb

```diff
@@ -32,6 +33,15 @@ def mutate_body
             end
           end
 
+          # Emit `Set.new`
+          #
+          # @return [undefined]
+          def emit_empty_set
+            return unless children.empty?
```

Comment:
Feel free to invite your team into the discussion. OT: did you ment to reply to this part of the thread?